__tests__/error-handlers.tests.js

```diff
@@ -76,4 +77,35 @@ describe('checkIsUser()', () => {
             })
         }
     })
+});
+
+describe('checkVotes()', () => {
+    it('should reject if desired decrement > votes count', async () => {
```

Comment:
Is the test in the articles.test.js not testing this ?

Reply:
It is! I thought we had to unit test our helper/error handling functions too - is it okay to skip that? 😊